lua/indent_blankline/init.lua

```diff
@@ -143,6 +145,11 @@ local refresh = function()
     }
 
     local max_indent_level = v("indent_blankline_indent_level")
+    local max_indent_increase = v("indent_blankline_max_indent_increase")
+    if max_indent_increase <= 0 then
+        max_indent_increase = max_indent_level
+    end
```

Comment:
instead of this, can you just make the default for `indent_blankline_max_indent_increase = indent_blankline_indent_level`

Reply:
Won't this require users to change their config if they use a non-default `indent_level` value?

Reply:
Same with what you have currently too. Setting the default as `0` and then changing it here, or setting the default as `max_indent_level` directly, doesn't make a difference. Just makes the code easier.